Best-Practices/Performance.md

```diff
@@ -20,10 +20,10 @@ This is an important area for people in the PowerShell community. While everyone
 For example:
 
 ```PowerShell
-$content = Get-Content file.txt
+$content = Get-Content -Path file.txt
```

Comment:
I don't think this makes this more readable, do you? I this it makes things more verbose, but doesn't improve or clarify anything.

Personally, I am opposed to explicit parameter names which don't improve the readability of the code.

For what it's worth, I think we agreed elsewhere (and implemented in PSScriptAnalyzer) to recommend only requiring parameter names when there are many parameters (PSScriptAnalyzer implemented this as "more than two").

Reply:
In my scripting, I always use explicit parameters so that it's always clear for other readers, but I'm on a team with many juniors at this point, so I guess it depends on the target audience for the document.